src/Randomizer.Data/Options/ItemCancelBehavior.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Randomizer.Data.Options;
+
+/// <summary>
+/// Enum for the different behaviors for pressing the item cancel button in SM
+/// </summary>
+public enum ItemCancelBehavior
+{
+    Vanilla,
```

Comment:
I'd add some descriptions here too, for completeness

Reply:
Do you mean comments or description attributes? Because unfortunately description attributes don't appear to be used by the EnumBindingSourceExtension.

Reply:
nvm, I was able to get that to work, so I added descriptions. You would really think WPF would have better support for enum + combo boxes natively.